swift/looker/Package.swift

```diff
@@ -16,17 +16,21 @@ let package = Package(
     dependencies: [
         // Dependencies declare other packages that this package depends on.
         // .package(url: /* package url */, from: "1.0.0"),
+      .package(
+           url: "https://github.com/Flight-School/AnyCodable",
```

Comment:
I don't know what the good ones are, but I assume this package is under a license we can use?

Reply:
Yes, it's MIT. I'll get the dependency notification going today.